anime_downloader/util.py

```diff
@@ -16,6 +16,7 @@
 from anime_downloader import session
 from anime_downloader.sites import get_anime_class, helpers
 from anime_downloader.const import desktop_headers
+from anime_downloader.config import Config
```

Comment:
Importing config in utils causes circular import on generation of config, moving this to slugify works